Tests/PackageLoadingPerformanceTests/ManifestLoadingTests.swift

```diff
@@ -0,0 +1,72 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import XCTest
+
+import Basic
+import TestSupport
+import PackageLoading
+
+#if ENABLE_PERF_TESTS
+
+class ManifestLoadingPerfTests: XCTestCase {
+    let manifestLoader = ManifestLoader(resources: Resources())
+
+    func write(_ bytes: ByteString, body: (AbsolutePath) -> ()) {
+        mktmpdir { path in
+            let manifestFile = path.appending(component: "Package.swift")
+            try localFileSystem.writeFileContents(manifestFile, bytes: bytes)
+            body(manifestFile)
+        }
+    }
+
+    func testTrivialManifestLoading_X2() {
```

Comment:
oh for the overall test case time ? I thought .1 - 1 for each iteration by measure block